### DISCUSSION TITLE
What's the TabView alternative in CommunityToolkit/Maui?

### DISCUSSION THREAD

**Discussion by monster-echo** [CommunityToolkit/Maui#1306] [Q&A] [answered]
https://learn.microsoft.com/en-us/xamarin/community-toolkit/views/tabview

The TabView has not disappeared in the CommunityTooit/Maui, Is it in progress or banned?

**Comment by VladislavAntonyuk** [comment, marked as answer]
There is no alternative from CommunityToolkit.MAUI. You can try Xamarin CommunityToolkit. MauiCompat

You can track the status here: https://github.com/CommunityToolkit/Maui/issues/121

**Reply by monster-echo** [reply]
Thanks.
